hypervisor/arch/x86/ept.c

```diff
@@ -248,8 +211,9 @@ void ept_mr_modify(struct vm *vm, uint64_t *pml4_page,
 	struct vcpu *vcpu;
 	uint16_t i;
 
-	mmu_modify_or_del(pml4_page, gpa, size,
-			prot_set, prot_clr, PTT_EPT, MR_MODIFY);
+	dev_dbg(ACRN_DBG_EPT, "%s,vm[%d] gpa 0x%llx size 0x%llx\n", __func__, vm->vm_id, gpa, size);
+
+	mmu_modify_or_del(pml4_page, gpa, size, prot_set, prot_clr, PTT_EPT, vm->vm_id, MR_MODIFY);
 
```

Comment:
vm_id should be last parameter?  It may be replaced with a callback handler.

At last, for HV page table, it is not vm_id.